src/containers/artist-recommendations/ArtistRecommendationsDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useRef } from 'react'
+
+import { useSpring, animated } from 'react-spring'
+
+import { useFlag } from 'containers/remote-config/hooks'
+import { FeatureFlags } from 'services/remote-config'
+
+import {
+  ArtistRecommendations,
+  ArtistRecommendationsProps
+} from './ArtistRecommendations'
+import styles from './ArtistRecommendationsDropdown.module.css'
+
+type ArtistRecommendationsDropdownProps = Omit<
+  ArtistRecommendationsProps,
+  'ref' | 'className' | 'itemClassName'
+> & {
+  isVisible: boolean
+}
+
+export const ArtistRecommendationsDropdown = (
+  props: ArtistRecommendationsDropdownProps
+) => {
+  const { isEnabled, isLoaded } = useFlag(
+    FeatureFlags.ARTIST_RECOMMENDATIONS_ENABLED
+  )
+  const { isVisible } = props
+  const childRef = useRef<HTMLElement | null>(null)
+
+  const rect = childRef.current?.getBoundingClientRect()
+  const childHeight = rect ? rect.bottom - rect.top : 0
+
+  const spring = useSpring({
```

Comment:
Can we steal a transition from here:
https://github.com/AudiusProject/stems/blob/1c74930ec1614cfbc935bba8a13e939365228884/src/utils/transitions.ts

Reply:
We should probably export those better - should I do an ugly import like `import { fast } from '@audius/stems/dist/utils/transitions'` in the meantime or just copy it over

Reply:
I support making the small stems change now if you want to! we can just merge it in super quickly & publish to npm. the ugly import is better than copying the code probably

Reply:
jk that doesn't work, I'll copy it

Reply:
b/c of the rollup config probs for stems.

we also have these for the now playing drawer
https://github.com/AudiusProject/audius-client/blob/master/src/containers/now-playing/NowPlayingDrawer.tsx#L48

copying in place seems good w/ me